Sources/RawRepresentable+Arbitrary.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  RawRepresentable+Arbitrary.swift
+//  SwiftCheck
+//
+//  Created by Brian Gerstle on 5/4/16.
+//  Copyright © 2016 Robert Widmann. All rights reserved.
```

Comment:
Copyright TypeLift.  If you spot that anywhere else, could you change it there too?